django/contrib/admin/apps.py

```diff
@@ -10,6 +10,7 @@ class SimpleAdminConfig(AppConfig):
     default_site = 'django.contrib.admin.sites.AdminSite'
     name = 'django.contrib.admin'
     verbose_name = _("Administration")
+    model_default_pk = 'django.db.models.AutoField'
```

Comment:
For now, yes. We're in a bit of a pickle here because if we don't set this explicitly then there will be a large and annoying amount of migrations (or `AppConfig`'s) that need to be created by every user who wishes to use `BigAutoField`.

The migrations we currently ship create models with `AutoField`'s, so and we have no good story for changing that. I think a likely outcome of this MR is that third party apps that ship migrations (including `contrib` apps) will need to explicitly define their primary keys. Which could be spun as a good thing, as it forces authors to think about this rather than just going with `AutoField`.